README.md

```diff
@@ -19,19 +19,24 @@ module "admin_tier" {
   github_team                 = "${var.github_team}"
   instance_type               = "${var.instance_type}"
   vpc_id                      = "${var.vpc_id}"
-  security_groups             = ["${var.security_groups}"]
+  name                        = "admin"
+  namespace                   = "${var.namespace}"
+  stage                       = "${var.stage}"
   subnets                     = ["${var.subnets}"]
-  associate_public_ip_address = "${var.associate_public_ip_address}"
```

Comment:
Let's use explicit `associate_public_ip_address` because default `associate_public_ip_address=true` should be changed to `false`.